src/components/buttons/cancel-button/cancel-button.ts

```diff
@@ -0,0 +1,19 @@
+import {CANCEL_BTN_IMG} from '@constants';
+import {BaseButtonComponent} from '../base-button-component/base-button-component';
+
+export default class CancelButtonHtml extends BaseButtonComponent {
+  constructor(
+    customId = 'cancel-button',
+    customClassName = 'cancel-button-image',
+    customImageAlt = 'Cancel Icon',
+    customTargetId = 'game-control',
```

Comment:
use the same or similar interface with base button options.
```
new BaseButtonComponent(options);

new CancelButtonHtml(options);
```
if there are no difference, no need to add a constructor as this will adopt the base one.

This will give us a better idea on the button's contracts/api

Reply:
please apply to other buttons as well.

Reply:
@miguelccodev can you pls check again? thanks